nncf/tensorflow/quantization/quantizers.py

```diff
@@ -154,12 +177,17 @@ def setup_input_transformation(self, input_shape, input_type, input_name, layer)
         :param input_name: Input name.
         :param layer: Layer, where the Quantizer is registered.
         """
-        self._pre_processing_fn, self._post_processing_fn = \
-            self._make_transformation_fns(input_shape, input_type, input_name, layer)
-
-    def _make_transformation_fns(self, input_shape, input_type, input_name, layer):
         channel_axes = get_channel_axis(input_type, input_name, layer)
 
+        try:
+            self._pre_processing_fn, self._post_processing_fn = \
+                self._make_transformation_fns(input_shape, channel_axes)
+        except NotImplementedError as e:
+            raise NotImplementedError(f'Additional information: input type {input_type}, input name {input_name},'
```

Comment:
The error text should more explicitly state what went wrong exactly, not simply log the info.

Reply:
Thanks for the comment! I fully agree with you! This try-except statement adds the additional information for the `NotImplementedError` message from the `_make_transformation_fns()` method.